doc/source/development/maintaining.rst

```diff
@@ -0,0 +1,177 @@
+.. _maintaining:
+
+******************
+Pandas Maintenance
+******************
+
+This guide is for pandas' maintainers. It may also be interesting to contributors
+looking to understand the pandas development process and what steps are necessary
+to become a maintainer.
+
+The main contributing guide is available at :ref:`contributing`.
+
+Roles
+-----
+
+Pandas uses two levels of permissions: **triage** and **core** team members.
+
+Triage members can label and close issues and pull requests.
+
+Core team members can label and close issues and pull request, and can merge
+pull requests.
+
+GitHub publishes the full `list of permissions`_.
+
+Tasks
+-----
+
+Pandas is largely a volunteer project, so these tasks shouldn't be read as
+"expectations" of triage and maintainers. Rather, they're general descriptions
+of what it means to be a maintainer.
+
+* Triage newly filed issues (see :ref:`maintaining.triage`)
+* Review newly opened pull request
+* Respond to updates on existing issues and pull requests
+* Drive discussion and decisions on stalled issues and pull requests
+
+.. _maintaining.triage:
+
+Issue Triage
+------------
+
+The issue tracker is many people's first interaction with the pandas project itself,
+beyond just using the library. As such, we want it to be a welcoming, pleasant
+experience.
+
+Here's a typical workflow for triaging a newly opened issue.
+
+1. **Is the necessary information provided?**
+
+   Ideally reporters would fill out the issue template, but many don't.
+   If crucial information (like the version of pandas they used), is missing
+   feel free to ask for that and label the issue with "Needs info".
+
+2. **Is this a duplicate issue?**
+
+   We have many open issues. If a new issue is clearly a duplicate, label the
+   new issue as "Duplicate" assign the milestone "No Action", and close the issue
+   with a link to the original issue. Make sure to still thank the reporter, and
+   encourage them to chime in on the original issue, and perhaps try to fix it.
+
+   If the new issue provides relevant information, such as a better or slightly
+   different example, add it to the original issue as a comment or an edit to
+   the original post.
+
+3. **Is the issue minimal and reproducible**?
+
+   For bug reports, we ask that the reporter provide a minimal reproducible
+   example. See http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports
+   for a good explanation. If the example is not reproducible, or if it's
+   *clearly* not minimal, feel free to ask the reporter if they can provide
+   and example or simplify the provided one. Do acknowledge that writing
+   minimal reproducible examples is hard work. If the reporter is struggling,
+   you can try to write one yourself and we'll edit the original post to include it.
```

Comment:
Something that I regularly do (and find valuable, also for afterwards writing simple tests), is further simplifying the example. Even if reporters provide a reproducible case, it can often be simplified a lot.